doc/writing.md

```diff
@@ -135,9 +134,8 @@ syntax, where `<icon name>` should be replaced with an actual icon name (_e.g._,
 :fa-<icon name>:
 ```
 
-You can find the icon names at: http://fontawesome.io/icons/. Those icons are
-scalable, which means you can use them anywhere (from titles to basic
-paragraphs), and they will always be rendered nicely.
+For icon names follow [this](http://fontawesome.io/icons/). These are scalable
+icons which will be rendered nicely irrespective of their usage.
```

Comment:
Instead of "their usage", how about something more specific like "the size icon you use"?